src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,54 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
+    private long id;
+    private String name;
+    private String country;
+
+    public Manufacturer(long id, String name, String country) {
+        this.id = id;
+        this.name = name;
+        this.country = country;
```

Comment:
```suggestion
        this.id = id;
        this(name, country)
```

lets call one constructor from another